TeensyDmx.cpp

```diff
@@ -237,6 +237,23 @@ void TeensyDmx::setChannels(
     }
 }
 
+// I am sure there is some code problems here but I wanted to suggest this feature for you to implement better than I can.
+void TeensyDmx::setChannelByArray(const uint8_t* values[]){
```

Comment:
This code will write over all sorts of memory.  I think you only want the middle loop.  The code as it stands will not work and will break the structure around it.